torch/ao/quantization/pt2e/eval_utils.py

```diff
@@ -53,4 +103,5 @@ def _move_exported_model_to_eval(model: torch.fx.GraphModule):
     QAT users should call this before performing inference on the model.
```

Comment:
In L102, can you add batchnorm to the description?

Reply:
Thanks and changed.